kqueen/config/base.py

```diff
@@ -29,7 +29,7 @@ class BaseConfig:
     CLUSTER_OK_STATE = 'OK'
     CLUSTER_PROVISIONING_STATE = 'Deploying'
     CLUSTER_DEPROVISIONING_STATE = 'Destroying'
-    CLUSTER_RESIZING_STATE = 'Resizing'
+    CLUSTER_UPDATING_STATE = 'Updating'
```

Comment:
You need to do corresponding change in the UI as well, so both API and UI work with the same default states

Reply:
done